pyproject.toml

```diff
@@ -44,11 +44,14 @@ classifiers = [
 dependencies = [
     "biopython",
     "python-redmine",
+    "unidecode",
```

Comment:
```suggestion
```
Not sure this import is needed (see below).

Reply:
It is used in the rnaseq.py module